src/hotspot/os/windows/memMapPrinter_windows.cpp

```diff
@@ -205,8 +208,8 @@ class MappingInfoPrinter {
     //            0         1         2         3         4         5         6         7         8         9         0         1         2         3         4         5         6         7
     //            012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789
     //            0x00007ffd5bd64000-0x00007ffd5c250000      5160960 r---  c-img 0x10d4000 C:\work\jdk\build\work-fastdebug\jdk\bin\server\jvm.dll
-    st->print_cr("from               to                        vsize prot  state    offset vm info/file");
-    st->print_cr("=============================================================================================================================");
+    st->print_cr("from               to                 prot  state    offset vminfo/file");
+    st->print_cr("=======================================================================");
```

Comment:
I think the extended "===" in the header was okay :-)

Reply:
I stuck a couple extra '===' in, just for you.  I do think it looks a bit better.